### PR TITLE
Architecture Updates

### PR DESCRIPTION
**Pull request by Yung-Beef**
swapped two lines

### CODE REVIEW

File: docs/lightpaper/architecture/Moderation.md [hunk 1]
```diff
@@ -12,7 +12,8 @@ inappropriate content.
 A single set of moderation rules like Twitter is too rigid and pushes all users to adopt the same
 perspectives in order to avoid being censored. Moderation per space means **“your space, your rules.”**
 These rules are handled by the blockchain. In addition, it is possible to modify the UI (user interface) to allow
-for greater control over the content displayed.
+for greater control over the content displayed. For exapmle, a social network focused on dogs would not show any 
+Subsocial content about cats.
```
Comment (F3Joule) [inline docs/lightpaper/architecture/Moderation.md:16]: How formal this lightpaper should be? :thinking:

File: docs/lightpaper/architecture/PostsAndSpaces.md [hunk 2]
```diff
@@ -31,15 +31,15 @@ using Subsocial?
 - **Airbnb** – apartment listings are spaces, while reviews are posts.
 - **Quora** – a question is a post, answers are comments/replies.
 - **Stack Overflow** –
-  - Answer writers can be rewarded, e.g. by the question writer.
-  - The distribution of payouts can also be split between what the question owner
+  - Answers can be rewarded, e.g. by the question asker.
+  - The distribution of payouts can also be split between what the question asker
```
Comment (F3Joule) [inline docs/lightpaper/architecture/PostsAndSpaces.md:35]: Isn't it a tautology here?

File: docs/lightpaper/architecture/Moderation.md (hunk 1 above)
Comment (Yung-Beef) [reply docs/lightpaper/architecture/Moderation.md:16]: Lol it is a simple and effective example

File: docs/lightpaper/architecture/PostsAndSpaces.md (hunk 2 above)
Comment (Yung-Beef) [reply docs/lightpaper/architecture/PostsAndSpaces.md:35]: I had to look that word up, congratulations ser. But no, I think those are two different things. Line 34 indicates that the question asker can reward answers, with full control, while line 35 shows that there could be some governance mechanism that adds some decentralization, so the question asker still gets to choose, but not with the entire reward basket.

**Review by F3Joule — APPROVED** [review]

**Review by siman — APPROVED** [review]

**Pull request merged**